docs/reference/hip_runtime_api_reference.rst

```diff
@@ -0,0 +1,33 @@
+.. meta::
+  :description: HIP runtime API reference page
+  :keywords: AMD, ROCm, HIP, CUDA, HIP runtime API, HIP runtime
+
+.. _runtime_api_reference:
+
+*******************************************************************************
+HIP runtime API
+*******************************************************************************
+
+This is the full HIP Runtime API reference. 
+
+:doc:`Modules <hip_runtime_api/modules>`
```

Comment:
The structure here seems kind of redundant. We have the Modules listed on the HIP Runtime API page (here), on the Modules List page, and as individual Modules listed below the Modules list. Seems like we could eliminate one of these layers. UI would suggest either the top-level HIP API page mentions and describes the four subcategories, but leaves the listing to those pages, or lists them, and eliminates the sub-pages.

Reply:
Solved as we discussed.